crezzur/PrestaShop
# Error Webservice combinations or categories on a blank schema

XML format requirements do not allow a tag to start with a number. With PHP 8+, an error is displayed that makes the web service partially unusable.

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      |  XML format requirements do not allow a tag to start with a number. With PHP 8+, an error is displayed that makes the web service partially unusable.
| Type?             | bug fix
| Category?         | WS 
| BC breaks?        | 
| Deprecations?     | 
| Fixed ticket?     | Fixes #32476 .
| Possible impacts? | The problem only occurs with **product combinations** and **categories** when using the parameter `schema=blank`.

**How to test?**
1. Make sure your PrestaShop web service is enabled
2. Create a new API key.
3. Using a web browser, go to the following URL **https://YOUR_API_KEY@YOUR_STORE_NAME/api/combinations?schema=blank** or for local host we can use **http://YOUR_API_KEY@127.0.0.1/YOUR_STORE_NAME/api/combinations?schema=blank**
I received comments regarding a problem that was supposed to be solved in version 8.1. However, after investigating, I could only find a pull request at this link: [GitHub Pull Request](https://github.com/PrestaShop/PrestaShop/pull/31023/files), and it does not seem to address the problem adequately.
